gufe/settings/models.py

```diff
@@ -3,130 +3,77 @@
-    scale: vdWScale
-    long_range_dispersion: Literal["isotropic"]
-    cutoff: FloatQuantity["angstrom"] = Field(9.0, description="Default units are angstroms")
-    switch_width: FloatQuantity["angstrom"] = Field(1.0, description="Default units are angstroms")
-    method: Literal["cutoff"]
-
-
-class ElectrostaticSettings(SettingsBaseModel):
-    """Settings for electrostatics.
-
-    See the `SMIRNOFF Electrostatics specification <https://openforcefield.github.io/standards/standards/smirnoff/#electrostatics>`_
-    for more details.
+class ThermoSettings(SettingsBaseModel):
+    """Settings for thermodynamic parameters.
 
+    No checking is done to ensure a valid thermodynamic ensemble is possible.
     """
 
-    # Tricky since this allows functions https://openforcefield.github.io/standards/standards/smirnoff/#electrostatics
-    periodic_potential: Union[Literal["Ewald3D-ConductingBoundary"], str]
-    nonperiodic_potential: Union[Literal["Coulomb"], str]
-    exception_potential: Union[Literal["Coulomb"], str]
-    scale: ElectrostaticScale
-    cutoff: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
-    switch_width: FloatQuantity["angstrom"] = Field(None, description="Default units are in angstroms")
-    solvent_dielectric: Optional[float]
+    temperature: FloatQuantity["kelvin"] = Field(
+        None, description="Simulation temperature, default units kelvin"
+    )
+    pressure: FloatQuantity["standard_atmosphere"] = Field(
+        None, description="Simulation pressure, default units standard atmosphere (atm)"
+    )
+    ph: Union[PositiveFloat, None] = Field(None, description="Simulation pH")
+    redox_potential: Optional[float] = Field(
+        None, description="Simulation redox potential"
+    )
 
 
-class GBSASettings(SettingsBaseModel):
-    """Settings for Generalized-Born surface area (GBSA) implicit solvent parameters.
+class ProtocolSettings(SettingsBaseModel, abc.ABC):
+    """Protocol-specific settings; this is a base class for protocol
+    developers to use for building any settings not included elsewhere.
 
-    See the `SMIRNOFF GBSA specification <https://openforcefield.github.io/standards/standards/smirnoff/#gbsa>`_
-    for more details.
     """
 
-    gb_model: str = "OBC1"
-    solvent_dielectric: float = 78.5
-    solute_dielectric: float = 1
-
+    ...
 
-class ForcefieldSettings(SettingsBaseModel):
-    """Settings for the forcefield.
-    """
 
-    # Metadata
-    date: Optional[date]
-    author: Optional[str]
+class BaseForcefieldSettings(SettingsBaseModel, abc.ABC):
+    """Base class for ForcefieldSettings objects"""
 
-    # These should also allow None
-    vdW: VdWSettings
-    electrostatics: ElectrostaticSettings
-    gbsa: GBSASettings
+    ...
 
 
-class ThermoSettings(SettingsBaseModel):
-    """Settings for thermodynamic parameters.
+class OpenMMSystemGeneratorFFSettings(BaseForcefieldSettings):
+    """This class is a WIP.
```

Comment:
does this need to be completed?

Reply:
This should be fine for now! Until we get another protocol forcefield application method/use case, this should be okay.